src/Circlereader/util/optionsAttribute.ts

```diff
@@ -0,0 +1,593 @@
+    let addOpt = new AddOptions();
+    addOpt = operator.builtinOptions<flatbuffers.Table>(addOpt);
+
+    attributes.push({
+      attribute: 'fusedActivationFunction',
+      value: ActivationFunctionType[addOpt.fusedActivationFunction()]
+    });
+  }
+
+  static getL2NormAttr(operator: Operator, attributes: Array<NodeAttributes>) {
+    let l2NormOpt = new L2NormOptions();
+    l2NormOpt = operator.builtinOptions<flatbuffers.Table>(l2NormOpt);
+
+    attributes.push({
+      attribute: 'fusedActivationFunction',
+      value: ActivationFunctionType[l2NormOpt.fusedActivationFunction()]
+    });
+  }
+
+  static getLocalResponseNormalizationAttr(operator: Operator, attributes: Array<NodeAttributes>) {
+    let localResponseNomalizationOpt = new LocalResponseNormalizationOptions();
+    localResponseNomalizationOpt =
+        operator.builtinOptions<flatbuffers.Table>(localResponseNomalizationOpt);
+
+    attributes.push({attribute: 'radius', value: localResponseNomalizationOpt.radius()});
+    attributes.push({attribute: 'bias', value: localResponseNomalizationOpt.bias()});
+    attributes.push({attribute: 'alpha', value: localResponseNomalizationOpt.alpha()});
+    attributes.push({attribute: 'beta', value: localResponseNomalizationOpt.beta()});
+  }
+
+  static getLSTMAttr(operator: Operator, attributes: Array<NodeAttributes>) {
+    let lstmOpt = new LSTMOptions();
+    lstmOpt = operator.builtinOptions<flatbuffers.Table>(lstmOpt);
+
+    attributes.push({
+      attribute: 'fusedActivationFunction',
+      value: ActivationFunctionType[lstmOpt.fusedActivationFunction()]
+    });
+    attributes.push({attribute: 'cellClip', value: lstmOpt.cellClip()});
+    attributes.push({attribute: 'projClip', value: lstmOpt.projClip()});
+    attributes.push({attribute: 'kernelType', value: LSTMKernelType[lstmOpt.kernelType()]});
+    attributes.push(
+        {attribute: 'asymmetricQuantizeInputs', value: lstmOpt.asymmetricQuantizeInputs()});
+  }
+
+  static getResizeBilinearAttr(operator: Operator, attributes: Array<NodeAttributes>) {
+    let resizeBilinearOpt = new ResizeBilinearOptions();
+    resizeBilinearOpt = operator.builtinOptions<flatbuffers.Table>(resizeBilinearOpt);
+
+    attributes.push({attribute: 'alignCorners', value: resizeBilinearOpt.alignCorners()});
+    attributes.push({attribute: 'halfPixelCenters', value: resizeBilinearOpt.halfPixelCenters()});
+  }
+
+  static getCallAttr(operator: Operator, attributes: Array<NodeAttributes>) {
+    let callOpt = new CallOptions();
+    callOpt = operator.builtinOptions<flatbuffers.Table>(callOpt);
+
+    attributes.push({attribute: 'subgraph', value: callOpt.subgraph()});
+  }
+
+  static getReshapeAttr(operator: Operator, attributes: Array<NodeAttributes>) {
+    let reshapeOpt = new ReshapeOptions();
+    reshapeOpt = operator.builtinOptions<flatbuffers.Table>(reshapeOpt);
+
+    attributes.push({attribute: 'newShapeArray', value: reshapeOpt.newShapeArray()});
```

Comment:
Although it is accessed by `newShapeArray`, original option name is `newShape` :)

```suggestion
    attributes.push({attribute: 'newShape', value: reshapeOpt.newShapeArray()});
```

Reply:
OK! Thank u for ur review 👍🏼